swatch-system-conduit/src/main/java/org/candlepin/subscriptions/conduit/security/ApiSecurityConfiguration.java

```diff
@@ -96,7 +96,7 @@ public class ApiSecurityConfiguration {
         "/**/version",
         "/api-docs/**",
         "/webjars/**",
-        "/api/swatch-system-conduit/internal/swagger-ui"
+        "/api/swatch-system-conduit/internal/swagger-ui/index.html"
```

Comment:
Redirect isn't happening at `/api/swatch-system-conduit/internal/swagger-ui` because it was getting caught up trying to authenticate `/swagger-ui` first.

To make it match the other ApiSecurityConfiguration we can add this:

```suggestion
        "/**/*spec.yaml",
        "/**/swagger-ui",
        "/**/swagger-ui/index.html",
```
The other option is to explicitly list both paths here

        "/api/swatch-system-conduit/internal/swagger-ui",
        "/api/swatch-system-conduit/internal/swagger-ui/index.html"